mautic/mautic
# MAUT-3455 New report's columns are not defaulted to the first data source

https://github.com/mautic/mautic/issues/8520
https://backlog.acquia.com/browse/MAUT-3455

Steps to test:
1. Go to Reports tab
2. Click New to add a new report
3. Without changing the source, go right to the data tab and see that the columns select list has been populated.
4. On the details tab, make sure the names in the Sources select list look right (i.e not the lowercase table names)
